hwtracer/src/collect/mod.rs

```diff
@@ -3,7 +3,7 @@
 use crate::{errors::HWTracerError, Trace};
```

Comment:
The commit message says something about "yktcare". Is that a typo?

Reply:
Oops, should be `yktrace`!

Reply:
OK, make sure we fix that at squash time.